Detectors/MUON/MCH/Simulation/src/Digitizer.cxx

```diff
@@ -140,7 +140,7 @@ int Digitizer::processHit(const Hit& hit, int detID, double event_time)
 
   bool padexists = seg.findPadPairByPosition(localX, localY, padidbendcent, padidnoncent);
   if (!padexists) {
-    LOG(ERROR) << "Did not find  _any_ pad for localX,Y=" << localX << "," << localY << ", for detID: " << detID;
+    LOG(WARNING) << "Did not find  _any_ pad for localX,Y=" << localX << "," << localY << ", for detID: " << detID;
```

Comment:
this is not sufficient. You also need to remove the return statement

Reply:
Hi @sawenzel :
sorry for my ignorance, but why is it not ok to return 0 for this function? That avoids to process any digit for this hit, which is the behaviour that is wanted as far as I understand.

Reply:
Then I don't see how this should fix the digitization problem. LOG(ERROR) alone did not crash the digitizer .. it is merely a different notification level.

Reply:
@sawenzel:
ok, I will see what I don't understand in the logic downstream that causes the problem.

Reply:
In fact, this does seem to work. I just tested it. But I am puzzled why "ERROR" leads to an abort in this process. Usually it doesn't. I'll merge it.